industry_real_estate/__manifest__.py

```diff
@@ -26,6 +26,7 @@
         'data/ir_model_2.xml',
         'data/crm_stages.xml',
         'data/res_config_setting.xml',
+        'data/sale_subscription_plan.xml',
```

Comment:
Well spotted! But the record defined inside is not used. I think it should replaced the one that is called on sale orders defined in demo (we should not rely on the ones from sale_subscription)